commons/src/main/java/org/eclipse/kapua/commons/util/xml/JAXBContextProvider.java

```diff
@@ -12,17 +12,17 @@
  *******************************************************************************/
 package org.eclipse.kapua.commons.util.xml;
 
-import javax.xml.bind.JAXBContext;
-
 import org.eclipse.kapua.KapuaException;
+import org.eclipse.kapua.model.KapuaObjectFactory;
+
+import javax.xml.bind.JAXBContext;
 
 /**
  * Jaxb context provider service definition.
  *
  * @since 1.0
- *
  */
-public interface JAXBContextProvider {
+public interface JAXBContextProvider extends KapuaObjectFactory {
```

Comment:
JAXBContextProviders has been marked as KapuaObjectFactory in order to be retrieved via the KapuaLocator on the application's edges (e.g.: runnables like the console, rest services, consumers, etc.)